2015-07-17 17:02:42,843 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@15a191e] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - gaoguangjin, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:47)] 
2015-07-17 17:03:00,731 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@14b43d] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - gaoguangjin, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:47)] 
2015-07-17 17:03:25,743 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@270664] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - gaoguangjin, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:47)] 
2015-07-17 17:03:40,808 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@270664] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - gaoguangjin, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:47)] 
2015-07-17 17:04:09,604 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@1454f68] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - gaoguangjin, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:47)] 
2015-07-17 17:47:57,665 -219  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@187514a] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 17:59:09,788 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@1f2b0dd] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - wan, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 17:59:09,808 -20   [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@1f2b0dd] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 18:00:30,563 -172  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@187514a] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 18:00:45,429 -172  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@904497] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 18:01:23,756 -0    [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@39b27b] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhang, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 18:01:23,772 -16   [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@39b27b] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 18:01:31,610 -156  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@c7d960] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:59)] 
2015-07-17 18:01:45,632 -157  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@187514a] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:61)] 
2015-07-17 18:02:40,068 -173  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@904497] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:61)] 
2015-07-17 18:07:23,645 -156  [main] com.team.gaoguangjin.shiro.ShiroRoleDemo  ERROR - 执行错误Realm [org.apache.shiro.realm.text.IniRealm@e9e799] was unable to find account data for the submitted AuthenticationToken [org.apache.shiro.authc.UsernamePasswordToken - zhangjinyue, rememberMe=false]. [com.team.gaoguangjin.shiro.ShiroRoleDemo.login(ShiroRoleDemo.java:61)] 
2015-07-20 10:29:56,132 -0    [main] com.team.gaoguangjin.shiro.FirstLoginDemo  ERROR - 认证错误！Authentication failed for token submission [org.apache.shiro.authc.UsernamePasswordToken - ggj, rememberMe=false].  Possible unexpected error? (Typical or expected login exceptions should extend from AuthenticationException). [com.team.gaoguangjin.shiro.FirstLoginDemo.login(FirstLoginDemo.java:59)] 
2015-07-22 09:20:31,668 -8810 [main] org.hibernate.engine.jdbc.spi.SqlExceptionHelper  ERROR - Cannot create PoolableConnectionFactory (Listener refused the connection with the following error:
ORA-12505, TNS:listener does not currently know of SID given in connect descriptor
The Connection descriptor used by the client was:
localhost:1521:ZSORA10G
) [org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(SqlExceptionHelper.java:146)] 
2015-07-22 09:49:46,295 -2818 [main] com.team.gaoguangjin.jdbc.hibernate.JunitHibernateInSpringTranstion  ERROR - ================= [com.team.gaoguangjin.jdbc.hibernate.JunitHibernateInSpringTranstion.findSpringManagerTransationQuery(JunitHibernateInSpringTranstion.java:61)] 
2015-07-22 09:49:46,357 -2880 [main] com.team.gaoguangjin.jdbc.hibernate.JunitHibernateInSpringTranstion  ERROR - For input string: "dd" [com.team.gaoguangjin.jdbc.hibernate.JunitHibernateInSpringTranstion.findSpringManagerTransationQuery(JunitHibernateInSpringTranstion.java:67)] 
2015-07-22 10:09:20,947 -11565 [main] org.hibernate.engine.jdbc.spi.SqlExceptionHelper  ERROR - ORA-01407: 无法更新 ("GAO"."TC_CLASSROOM"."ROOM_SIZE") 为 NULL
 [org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(SqlExceptionHelper.java:146)] 
2015-07-23 16:40:06,605 -0    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.test(ExceptionThrow.java:26)] 
2015-07-23 16:40:20,520 -0    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - 父类捕获错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.main(ExceptionThrow.java:18)] 
2015-07-23 16:46:12,027 -0    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - 父类捕获错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.main(ExceptionThrow.java:19)] 
2015-07-23 16:46:30,933 -0    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.test(ExceptionThrow.java:32)] 
2015-07-23 16:47:53,506 -0    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.test(ExceptionThrow.java:39)] 
2015-07-23 16:47:53,521 -15   [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - 父类捕获错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.main(ExceptionThrow.java:19)] 
2015-07-23 16:53:32,896 -0    [main] com.team.gaoguangjin.javabase.exception.FinallyReturn  ERROR - 出错了！For input string: "a" [com.team.gaoguangjin.javabase.exception.FinallyReturn.getString(FinallyReturn.java:17)] 
2015-07-23 16:55:45,952 -0    [main] com.team.gaoguangjin.javabase.exception.FinallyReturn  ERROR - 出错了！For input string: "a" [com.team.gaoguangjin.javabase.exception.FinallyReturn.getString(FinallyReturn.java:18)] 
2015-07-23 16:56:06,020 -0    [main] com.team.gaoguangjin.javabase.exception.FinallyReturn  ERROR - 出错了！For input string: "a" [com.team.gaoguangjin.javabase.exception.FinallyReturn.getString(FinallyReturn.java:18)] 
2015-07-23 17:06:17,524 -0    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.test(ExceptionThrow.java:70)] 
2015-07-23 17:06:17,528 -4    [main] com.team.gaoguangjin.javabase.exception.ExceptionThrow  ERROR - 父类捕获错误 [com.team.gaoguangjin.javabase.exception.ExceptionThrow.main(ExceptionThrow.java:19)] 
2015-07-23 17:08:08,065 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:40)] 
2015-07-23 17:08:08,069 -4    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - 父类捕获错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.main(ExceptionThrowTypeTwo.java:19)] 
2015-07-23 17:08:28,137 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:40)] 
2015-07-23 17:09:31,534 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:41)] 
2015-07-23 17:10:08,477 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:38)] 
2015-07-23 17:30:01,822 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:38)] 
2015-07-23 17:33:04,649 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误 [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:38)] 
2015-07-23 17:35:32,173 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 17:37:54,664 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:54)] 
2015-07-23 17:37:54,858 -194  [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 17:39:09,160 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:54)] 
2015-07-23 17:39:09,318 -158  [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 17:41:13,142 -74   [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:49)] 
2015-07-23 17:41:13,182 -114  [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:38)] 
2015-07-23 17:42:46,828 -58   [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:49)] 
2015-07-23 17:42:46,857 -87   [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test(ExceptionThrowTypeTwo.java:38)] 
2015-07-23 17:51:14,621 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:37)] 
2015-07-23 17:51:14,692 -71   [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:48)] 
2015-07-23 18:00:00,956 -7    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:00:00,958 -9    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:00:15,322 -6    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:00:15,324 -8    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:01:26,015 -7    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:01:26,017 -9    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:01:48,605 -8    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:01:48,607 -10   [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:07:49,569 -6    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:07:49,570 -7    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:07:57,774 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:08:06,122 -6    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:08:06,124 -8    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:08:36,675 -0    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test3():aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test3(ExceptionThrowTypeTwo.java:52)] 
2015-07-23 18:08:36,682 -7    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - catch错误aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test1(ExceptionThrowTypeTwo.java:32)] 
2015-07-23 18:08:36,684 -9    [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test2()：aaa (系统找不到指定的文件。) [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test2(ExceptionThrowTypeTwo.java:43)] 
2015-07-23 18:08:36,685 -10   [main] com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo  ERROR - test5 catch错误For input string: "a" [com.team.gaoguangjin.javabase.exception.throwexception.ExceptionThrowTypeTwo.test6(ExceptionThrowTypeTwo.java:70)] 
2015-07-27 13:26:41,112 -22   [main] com.team.gaoguangjin.shiro.TestSecond  ERROR - 认证失败：null [com.team.gaoguangjin.shiro.TestSecond.test(TestSecond.java:36)] 
